playbooks/10 - SP - GDPR Framework/Get Updated Data Breach Report from DPO.json

```diff
@@ -581,14 +581,14 @@
                     "mitigationActions": "{{vars.steps.Update_DPO_Report.input['whatMitigationActionsHaveBeenImplemented?']}}",
                     "noOfAffectedUsers": "{{vars.steps.Update_DPO_Report.input['no.OfAffectedUsers']}}",
                     "whereDataWasHosted": "{{vars.steps.Update_DPO_Report.input['whereDataWasHosted?']}}",
-                    "wasTheDataEncrypted": "{{\"ChooseYesOrNo\" | picklist(vars.steps.Update_DPO_Report.input['wasTheDataEncrypted?'], \"@id\")}}",
+                    "wasTheDataEncrypted": "{{\"YesNoInput\" | picklist(vars.steps.Update_DPO_Report.input['wasTheDataEncrypted?'], \"@id\")}}",
                     "whatWasTheDataFormat": "{{vars.steps.Update_DPO_Report.input['whatWasTheDataFormat?']}}",
                     "whatWasTheDataSource": "{{vars.steps.Update_DPO_Report.input['whatWasTheDataSource?']}}",
                     "repercussionsOfBreach": "{{vars.steps.Update_DPO_Report.input['whatAreTheRepercussionsOfBreach?']}}",
-                    "wasTheExposureResolved": "{{\"ChooseYesOrNo\" | picklist(vars.steps.Update_DPO_Report.input['wasTheExposureResolved?'], \"@id\")}}",
+                    "wasTheExposureResolved": "{{\"YesNoInput\" | picklist(vars.steps.Update_DPO_Report.input['wasTheExposureResolved?'], \"@id\")}}",
```

Comment:
How about "isExposureResolved"

Reply:
This is a standard GDPR Data Breach Notification Template, all questions are in the past tense. And refers to the time when a data breach has happened instead of the current state

Reply:
Resolved denotes past tense. Was is not needed here. Please check GDPR docs once